modules/ismp/clients/grandpa/src/benchmarking.rs

```diff
@@ -0,0 +1,50 @@
+#![cfg(feature = "runtime-benchmarks")]
+
+use super::*;
+use frame_benchmarking::v2::*;
+use frame_system::RawOrigin;
+use ismp::host::StateMachine;
+
+#[benchmarks(where T: Config)]
+mod benchmarks {
+    use super::*;
+
+    #[benchmark]
+    fn add_state_machines() -> Result<(), BenchmarkError> {
```

Comment:
shouldn't this benchmark be dynamic? so that the weight is calculated based on the number of items to be added?